paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/reporting/ImagePanelRenderer.kt

```diff
@@ -0,0 +1,61 @@
+package app.cash.paparazzi.gradle.reporting
+
+import org.gradle.reporting.ReportRenderer
+
+internal class ImagePanelRenderer : ReportRenderer<List<ScreenshotDiffImage>, SimpleHtmlWriter>() {
+  override fun render(image: List<ScreenshotDiffImage>, htmlWriter: SimpleHtmlWriter) {
+    // Wrap in a <span>, to work around CSS problem in IE
```

Comment:
side question, isn't IE dead now? Does Edge have the same rendering issues that need the workaround?

Reply:
this is copied over from the actualy impl from gradle. I tend to agree with you but for now think we can keep this in.